summary/types/glucose.go

```diff
@@ -162,31 +162,38 @@ func (rs *GlucoseRanges) Finalize(days int) {
 	}
 }
 
-func (rs *GlucoseRanges) Update(record *glucoseDatum.Glucose) {
-	normalizedValue := *glucose.NormalizeValueForUnits(record.Value, record.Units)
+func (rs *GlucoseRanges) Update(record *glucoseDatum.Glucose) error {
+	classification, err := glucoseDatum.TidepoolADAClassificationThresholdsMmolL.Classify(record)
```

Comment:
I think you. need to replace the ["config" values in the summary](https://github.com/tidepool-org/platform/blob/master/summary/types/summary.go#L23-L27) with the new thresholds

Reply:
My fix for this is somewhat ... awkward maybe? I welcome other ideas, but I think this is going to all be changing soonish with the introduction of alternate ranges, so I'm not sure it's worth spending a ton of time on.